joblib/numpy_pickle.py

```diff
@@ -360,18 +269,20 @@ def dump(value, filename, compress=0, cache_size=100, protocol=None):
         The object to store to disk
     filename: string
         The name of the file in which it is to be stored
-    compress: integer for 0 to 9, optional
-        Optional compression level for the data. 0 is no compression.
+    compress: integer from 0 to 9 or boolean, optional
```

Comment:
I tend to use Python types in the first line of the parameter doc, e.g. bool rather than boolean, str rather than strings. Maybe just me.